scripts/src/public/app.ts

```diff
@@ -32,6 +32,7 @@ export const app: express.Express = createApp();
 // routes
 createRoutes(app, "/v2");
 createV1Routes(app, "/v1");
+createPublicFacing(app, "/partners/v1");
```

Comment:
I don't like that the version is not the first thing in the URL. You can just use `/v1/applications/<>/...` or `v2`